acceptance/tests/environment/environment_scenario-bad.rb

```diff
@@ -16,32 +16,32 @@
 apply_manifest_on(master, environment_manifest(testdir), :catch_failures => true)
```

Comment:
@er0ck This wasn't the test that was failing for us, it was `environment_scenario-non_existent.rb`.